.gitignore

```diff
@@ -16,3 +16,6 @@ logs
 results
 
 npm-debug.log
+
+## WebStorm IDE
+.idea
```

Comment:
Global ignore rules (such as files created by editors) belong in your personal git ignore config file, not in the project's.